python/nnvm/frontend/mxnet.py

```diff
@@ -358,8 +358,13 @@ def from_mxnet(symbol, arg_params=None, aux_params=None):
         sym = symbol(data)
         sym = _from_mxnet_impl(sym, {})
         params = {}
-        for k, v in symbol.collect_params().items():
-            params[k] = tvm.nd.array(v.data().asnumpy())
+        for block in symbol._children:
```

Comment:
This does not fix nested blocks for gluon models. Also, symbols are not fixed by this hack.
I think we should add a transpose op on top of weights inside Deconvolution Op.